tsconfig.base.json

```diff
@@ -8,10 +8,10 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "importHelpers": true,
-    "target": "es2015",
+    "target": "es2021",
```

Comment:
Change of `target` is pretty significant and should at least be committed separately, but if not needed here (which I think it isn't), then do in separate PR.

Reply:
I'd like to increase the `lib`, but increase of `target` depends on browser support, which is a question for product owners.